playbooks/create_uda_job.yml

```diff
@@ -0,0 +1,20 @@
+#=> Create a Cohesity protection job for UDA hosts.
+---
+- hosts: localhost
+  gather_facts: no
+  collections:
+    - cohesity.dataprotect
+  tasks:
+    - name: "Uda Protection job"
+      cohesity_uda_job:
+        state: present
+        job_name: protect_uda
+        storage_domain: "DefaultStorageDomain"
+        policy: "Silver"
+        endpoint: "x.x.x.x"
+        cancel_active: true
+        objects:
+          - object
+        cluster: "{{ cohesity_server }}"
```

Comment:
Make this part of inventory instead of the playbook

Reply:
We have the declared cohesity_server variable in the inventory file, we will refer the variable here.